recipe/meta.yaml

```
@@ -1,23 +1,28 @@
 {% set name = "numcodecs" %}
-{% set version = "0.12.1" %}
+{% set version = "0.13.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 05d91a433733e7eef268d7e80ec226a0232da244289614a8f3826901aec1098e
+  sha256: ba4fac7036ea5a078c7afe1d4dffeb9685080d42f19c9c16b12dad866703aa2e
+  patches:
+    # Fix the way assembly files are included on x86_64
+    - patches/including_assmembly.patch
 
 build:
-  number: 1
+  number: 0
   script_env:
```

To make it easier to update the `skip`, would recommend keeping it here near `number`. The other lines below don't typically change

```suggestion
  number: 0
  skip: true  # [py<310]
  script_env:
```